### PR TITLE
[Doc][Kommander] Updated RN and removed beta: true flags

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Description of changes being made
Updated RN and removed beta: true flags

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ enterprise: false
 
 <!-- markdownlint-disable MD034 -->
 
-**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 1 was released on March 11, 2021.**
+**D2iQ&reg; Kommander&reg; version 1.4.0 was released on April 29, 2021.**
```
Comment (msdolbey) [inline pages/dkp/kommander/1.4/release-notes/index.md:13]: I think we're going to update this to May 3?
```suggestion
**D2iQ&reg; Kommander&reg; version 1.4.0 was released on May 3, 2021.**
```

**Comment by cmastr** [comment]
We need to add a section to the release notes that talks about certified catalogs:
https://docs.google.com/presentation/d/13-jtu8MgmZuxq4bJBJJz8ZnWFfXArD-B-4OHFvfS5TA/edit#slide=id.gd3834abdb0_0_12 

Use this slide deck for info.

**Comment by gracedo** [comment]
Updated release notes in rc.3: https://github.com/mesosphere/kubeaddons-kommander/releases/tag/v1.4.0-rc.3

File: pages/dkp/kommander/1.4/release-notes/index.md [hunk 2]
```diff
@@ -21,6 +21,37 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
...
+For more information on this capability, see [Attach an Existing Kubernetes Cluster](../clusters/attach-cluster)
+
+## Catalog Workload Certification
+
+D2iQ now certifies workloads for use with Kommander. All workloads that have been tested and certified for successful configuration and provisioning on Konvoy are flagged in Kommander with a certification icon. For more information, see [Project Platform Services](../projects/platform-services/).
+
+## Helm charts
+The Projects tab now shows all of the current Helm Release charts, their chart version, and the names of the clusters. For more information, see [Project Deployments](../projects/project-deployments/).
+
 # Breaking changes
```
Comment (cmastr) [inline pages/dkp/kommander/1.4/release-notes/index.md:56]: Need to add COPS tickets to fixes below.

Comment (cneth) [reply pages/dkp/kommander/1.4/release-notes/index.md:56]: Here are the ones I am aware of:

COPS-6897: This one was filed by an internal customer and the 'fix' is described in the "Workspace permissions update" above.   Given that this was an internally raised issue, I see no reason to cite the COPS number here.

(COPS-6912) - This one needs to be referenced, as I know some field folks ran into this and so customers probably did as well.  See (D2IQ-75347) for details on this issue, but we don't need to reference that # in the RN. Suggested RN:  "Fixed a problem where sometimes valid licenses were briefly displayed as invalid."

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.4/release-notes/index.md (hunk 2 above)
Comment (cmastr) [reply pages/dkp/kommander/1.4/release-notes/index.md:56]: Went through your notes and ensured these were documented as you asked.

**Review by cneth — APPROVED** [review]

**Review by cmastr — APPROVED** [review]

**Pull request closed without merging**